av/video/frame.pyx

```diff
@@ -137,6 +137,9 @@ cdef class VideoFrame(Frame):
             count = self.format.ptr.comp[i].plane + 1
             if max_plane_count < count:
                 max_plane_count = count
+        if self.format.name == 'pal8':
+            max_plane_count = 2
+
```

Comment:
So... does `pal8` actually have 2 planes, or are we (ab)using `VideoPlane` as a way to expose the palette?

Reply:
`pal8` does have two planes, the second one is for storing the palette

Reply:
And the code above it won't result in `max_plane_count == 2`? That seems weird. Okay.

Reply:
It won't because the palette has no associated component, it's just metadata.